test/computed_test.dart

```diff
@@ -102,4 +102,21 @@ void main() {
     verify(context.nameFor('Computed'));
     verify(context.trackDerivation(value, fn));
   });
+
+  test('computed catches exception in evaluation', () {
+    var shouldThrow = true;
+
+    final x = computed(() {
```

Comment:
Computed values in mobx re-throw exceptions thrown in the body. Will this come later?

Reply:
Yes. Currently we have something called the **Error Boundary** in the MobX parlance. This means errors are trapped locally to keep the MobX system consistent. We will introduce global option later to disable this. I think we are good to merge

Reply:
Please see the `disableErrorBoundaries` option in [this page](https://mobx.js.org/refguide/api.html)